package.json

```diff
@@ -44,9 +45,12 @@
     "clean": "rm -rf ./public/app/dist && rm -rf ./public/dist ",
     "bundle": "webpack -p --config ./public/webpack.config.js ",
     "bundle-dev": "webpack -d -w --config ./public/webpack.config.js",
+    "js-files": "find ./public \\( -name '*.js' -o -name '*.jsx' \\) ! -path './node_modules/*' ! -path './public/bower_components/*' ! -path './coverage/*' ! -path './public/dist/*' ! -path './jspm_packages'",
     "test": "npm run bundle && node ./node_modules/karma/bin/karma start ./public/app/project-setup/test/karma.conf.js --single-run",
     "tdd": "npm run bundle-dev && node ./node_modules/karma/bin/karma start ./public/app/project-setup/test/karma.conf.js",
     "build": "npm run clean && npm run bundle && cp -r ./public/app/* ./public/dist",
+    "eslint": "node_modules/.bin/eslint `npm run -s js-files`",
+    "lint": "npm run eslint",
     "precommit": "npm test"
```

Comment:
I thought I'd go over all the errors and fix them first... But it can be done since we're using `--no-verify` anyway.